elements/pfe-navigation/src/pfe-navigation.scss

```diff
@@ -156,14 +158,15 @@ pfe-icon {
     flex-shrink: 0; // Prevent flexbox from collapsing the padding.
     align-items: flex-start;
     justify-content: center;
-    min-width: #{pfe-local($region: logo, $cssvar: MinWidth)};
+    width: #{pfe-local($region: logo, $cssvar: MinWidth)};
+    flex: 0 0 #{pfe-local($region: logo, $cssvar: MinWidth)};
     
     padding-top: #{pfe-local(Padding--vertical)};
     padding-bottom: #{pfe-local(Padding--vertical)};
-    padding-right: 48px; // ensure gap between main menu links and logo
+    padding-right: 25px; // ensure gap between main menu links and logo
```

Comment:
Is it helpful to use variables for these values?
`padding-right: 25px;`
becomes
`padding-right: calc( #{pfe-var(container-spacer)} + 9px );`

Reply:
@starryeyez024 I made this change. Feel free to revert or change it.